gto-support-db/src/androidTest/java/org/ccci/gto/android/common/db/AbstractDaoTest.java

```diff
@@ -22,4 +24,102 @@ public void testGetTable() throws Exception {
         final TestDao dao = getDao();
         assertThat(dao.getTable(Root.class), is(RootTable.TABLE_NAME));
     }
+
+    @Test
+    public void testGetCursor() {
+        final TestDao dao = getDao();
+        final Cursor cursor = dao.getCursor(Query.select(Root.class));
+
+        assertThat(cursor.getColumnName(0), is("_id"));
```

Comment:
that way ordering of columns within the Cursor isn't being tested, but column presence is